app/views/home/index.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="flex flex-col items-center px-10 md:px-32 lg:px-60 py-10 lg:py-14">
   <h1 class="mb-7 text-primary">¡Hola!</h1>
-  <h5 class="mb-7">Te damos la bienvenida a</h5>
+  <h5 class="mb-8">Te damos la bienvenida a</h5>
+  <%= image_tag("logo.png", :class => "w-10/12 md:w-8/12 lg:w-6/12 mb-12") %>
```

Comment:
This can be merged into the previous commit, since it belongs to the home layout and there's no need for it to be in a single commit.

Reply:
It was add in another commit because I needed to install the Asset Pipeline in another Issue and PR and then rebase this branch with master, would you still recommend to merge it or will it mess up the commit history?

Reply:
If the Asset Pipeline PR has been merged in master, it should be fine to add the logo in the first commit. Let me know if you need help with this.